cpp/benchmarks/fixture/benchmark_fixture.hpp

```diff
@@ -29,9 +29,11 @@ namespace {
 // memory resource factory helpers
 inline auto make_cuda() { return std::make_shared<rmm::mr::cuda_memory_resource>(); }
 
-inline auto make_pool()
+inline auto benchmark_pool_instance()
```

Comment:
Since other functions have the prefix `make_`, please also use that prefix. So maybe `make_pool_instance` or something similar :)

Reply:
```suggestion
inline auto make_pool_instance()
```